src/apb/entrypoint.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
 """GitHub Action to rebuild respositories that haven't be built in a while."""
```

Comment:
It's not part of this update, but we might as well fix this typo:
`haven't be built` --> `haven't been built`

Reply:
Ah, good eye. Fixed in 50bcd3c.